docs/api-reference/tasks-api.md

```diff
@@ -25,77 +25,1448 @@ sidebar_label: Tasks
 
 This document describes the API endpoints for task retrieval, submission, and deletion for Apache Druid.
 
-## Tasks
-
 Note that all _interval_ URL parameters are ISO 8601 strings delimited by a `_` instead of a `/`
 as in `2016-06-27_2016-06-28`.
```

Comment:
```suggestion
For query parameters that take an interval, provide ISO 8601 strings delimited by `_` instead of `/`.
For example, `2023-06-27_2023-06-28`.
```